frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -340,6 +342,9 @@ Open the Console in your web browser and inspect the value of `window.SERVER_FLA
 list of dynamic plugins the Console loads at runtime. For local development, this should only
 include plugin(s) listed via `-plugins` Bridge argument.
 
+Console development builds allow you to interact with the `PluginStore` object that manages
+all plugins and their extensions directly in your web browser with `window.pluginStore`.
```

Comment:
Is this meant to be a public API or only for debugging? I think we should make it clear how this can/should be used.

Reply:
@spadgett `window.pluginStore` is available in Console development builds only, so it's not a public API.

This is similar to `window.store` (also available in Console development builds only) exposed at `public/redux.ts`.

Reply:
OK, if we're going to add it to the README, I think we want to make that clear.